src/building/suggested.md

```diff
@@ -161,12 +161,21 @@ git worktree add -b my-feature ../rust2 master
 You can then use that rust2 folder as a separate workspace for modifying
 and building `rustc`!
 
-## Building with system LLVM
+## Skipping LLVM Build
 
-By default, LLVM is built from source, and that can take significant amount of
-time.  An alternative is to use LLVM already installed on your computer.
+By default, LLVM is built from source, and that takes significant amount of
+time. One way to avoid that is to add this to `config.toml`:
 
-This is specified in the `target` section of `config.toml`:
+```toml
+[llvm]
+download-ci-llvm = true
+```
+
+Downloading LLVM from CI is still experimental though, and might not be
+available on all platforms. Otherwise, we'd make it a default!
```

Comment:
This does not feel like the right place to indicate planned policy/default, here referring to "we'd make it a default".

Reply:
This also was done in the interest of future compatibility. The thing is in flux, so I find it useful:

* to document it early, even if incomplete
* to be clear that there might be changes, and that the docs might be out of date
* to not be super strict with keeping the docs up to date.

Reply:
I was only referring to "Other, we'd make it a default". First sentence is all good.